core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci

```diff
@@ -61,3 +61,5 @@ gcr.io/istio-testing/build-tools:release-1.14-204316d1ee4013432e0b35d8ceba3a943e
 quay.io/maistra-dev/maistra-proxy-builder:2.0 registry.ci.openshift.org/ci/maistra-proxy-builder:2.0
 quay.io/maistra-dev/maistra-proxy-builder:2.1 registry.ci.openshift.org/ci/maistra-proxy-builder:2.1
 quay.io/almartin/odf-console-ci-runner:latest registry.ci.openshift.org/ci/odf-console-ci-runner:latest
+quay.io/strimzi-qe/strimzi-tests:release-0.29.x registry.ci.openshift.org/ci/strimzi-tests:release-0.29.x
+quay.io/strimzi-qe/strimzi-tests:release-0.32.x registry.ci.openshift.org/ci/strimzi-tests:release-0.32.x
```

Comment:
Before we merge this I'm just curious to know more about what goes into the creation and maintenance of these `quay.io/strimzi-qe/strimzi-tests` images.

Is there a repo with the source code used to create the image that you can share?
Is there CI verifying that changes to the source code isn't breaking these image?
Is there CI in place to push the latest changes to these images?

Reply:
Addressed these questions in this thread 
https://redhat-internal.slack.com/archives/C04QDE5TK1C/p1679413818120849?thread_ts=1678722379.723629&cid=C04QDE5TK1C